.github/workflows/cursor-pr-review-agent.yml

```
@@ -0,0 +1,276 @@
+            COMMENTS=$(./scripts/pr-check-comments.sh --pr="$PR_NUMBER" --review-batch="$REVIEW_ID" --limit=7)
+          fi
+          
+          # Save comments to a temporary file to avoid issues with multiline strings in env vars or args
+          echo "$COMMENTS" > /tmp/pr-review-comments.md
+          echo "comments_path=/tmp/pr-review-comments.md" >> $GITHUB_OUTPUT
+
+      - name: Compute variables
+        id: vars
+        env:
+          IS_BOT: ${{ steps.context.outputs.is_bot }}
+        run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2034:warning:2:1: REVIEW_ID appears unused. Verify use (or export if used externally)

(shellcheck)